### DISCUSSION TITLE
The app stopped working on M4 with 15.1. I

### DISCUSSION THREAD

**Discussion by micleeso** [MonitorControl/MonitorControl#1673] [General]
I was using the app, then tried BetterDisplay. I realized I preferred Monitor Control. I used it for a bit, closed it then the app stopped and it will not execute again. 

If I delete the app, then reinstall, it prompts to allow the app to run - the app doesn't appear. Click on the app same result

Is there a complete uninstall or is there something more happening?

Sequoia 15.1 (24B2083)
App version 4.3.3

Thank you for your help.

**Comment by waydabber** [comment]
Maybe the app menu is set to hidden? Do you see the app running in Activity Monitor?

**Reply by micleeso** [reply]
It is present in Activity monitor for 10 seconds then it terminates.
